eng/pipelines/templates/variables/globals.yml

```diff
@@ -20,3 +20,7 @@ variables:
 
   # disable code coverage redesign until it's been fixed (causes inadvertent deflation of CC numbers)
   GOEXPERIMENT: nocoverageredesign
+
+  # Supported versions for testing. These variables are referenced in test matrix files.
+  GO_VERSION_N0: 1.24.1
+  GO_VERSION_N1: 1.23.7
```

Comment:
Can we use these in the base matrix template or reference that template here so there's only one file to edit when a new Go ships?

Reply:
This is the one file, and it will propagate to the matrix configs, unless I missed a spot? Or were you thinking we'd have these version settings elsewhere?

Reply:
I'm not sure what I was getting at; I suppose I misread this PR or misunderstood matrix generation. This looks good to me